eq-author/src/components/RichTextEditor/LinkPlugin/index.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+import { EditorState, Modifier } from "draft-js";
+import Tooltip from "components/Forms/Tooltip";
+
+const ENTITY_TYPE = "LINK";
+const MUTABILITY = "IMMUTABLE";
+
+const filterConfig = {
+  type: ENTITY_TYPE,
+};
+
+const linkFromHTML = {
+  a: (nodeName, node, createEntity) =>
+    createEntity(ENTITY_TYPE, MUTABILITY, { url: node.href }),
+};
+
+const linkToHTML = {
+  [ENTITY_TYPE]: (entity, originalText) => (
+    <a href={entity.data.url} target="_blank" rel="noopener noreferrer">
+      {originalText}
+    </a>
+  ),
+};
+
+const Link = styled.a`
+  background: #d6e4ff;
+`;
```

Comment:
Hi Tom. Inside this file, `src/constants/theme.js` there's an object called `colors`. Are there any colours in there that you could use instead of this new one? 

You can speak to Joe if you're not sure -- iirc we had a conversation at a retrospective (before you joined the team I think - so you wouldn't know about this) about trying to reuse as many colours as we can to avoid adding new ones for single uses.

If it turns out none of them are appropriate, you can add your new colour into that object and reference it in the code that you're written. The benefit is that we can centralise our colours so that if we ever need to bulk change them we can do in a single place.

Reply:
Ah - cheers Matt. I'll take a look.

Reply:
`colors.paleBlue` is a pretty good fit; I've pushed the change now.